example/app-framework/wamr_config_wasmnizer_ts.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2023 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+set (WAMR_BUILD_PLATFORM "linux")
```

Comment:
We can make this as a selectable profile, just like that in WAMR sample, and we can provide several profiles (e.g. interp, aot, simple_libdyntype).